### PR TITLE
Images support multi arch

### PR DESCRIPTION
**Pull request by JaredTan95**
Fixes https://github.com/open-telemetry/opentelemetry-demo/issues/458

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -16,29 +16,65 @@ jobs:
...
+          - file: ./src/frontend/Dockerfile.cypress
+            tag_suffix: frontend-tests
+          - file: ./test/Dockerfile
+            tag_suffix: integrationTests
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - name: Update .env
```
Comment (austinlparker) [inline .github/workflows/release.yml:24]: Please bring this step back into the release automation

**Review by austinlparker — CHANGES_REQUESTED** [review]
lgtm but you need to bring the release tag automation back in.

**Review by austinlparker — APPROVED** [review]
lgtm!

**Review by julianocosta89 — APPROVED** [review]
LGTM

**Pull request merged**